css/style.css

```diff
@@ -1,3 +1,100 @@
+.container {
+	margin: 50px;
+	background-color: #ECEDEF;
+	font-family: sans-serif;
+}
+
+.photo-section {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+}
+
+.San-Fransisco {
+	background: url(../img/san-francisco.jpg);
+	
+}
+.New-York {
+	background: url(../img/new-york.jpg);
+	
+}
+
+.London {
+	background: url(../img/london.jpg);
+	
+}
+
+.Napa {
+	background: url(../img/Napa.jpg);
+	
+}
+
+.Sonoma {
+	background: url(../img/Sonoma.jpg);
+	
+}
+
+.San-Fransisco-two {
+	background: url(../img/san-francisco-2.jpg);
+	
+}
+
+
+.photo-format {
+	background-repeat: no-repeat;
+	margin: 10px;
+	height: 200px;
+	width: 200px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	background-size: 100%;
```

Comment:
It may not have been apparent, but the top row images were rectangular, and the second were square. When you specify `background-size: 100%` (this sets width 100% and height auto) it scales the width to fit and height is proportionally scaled after.

For square photos in a square div, the height and width are the same so everything looks to line up.

On landscape rectangles, however, it scales the image to fit by width, but the displayed height is now shorter than the containing div.

To get the behavior you want here, filling the div with your image but without squishing proportions, try `background-size: cover;`

More info on background size: https://css-tricks.com/almanac/properties/b/background-size/